core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -266,8 +267,16 @@ object Boot extends IOApp with LazyLogging {
       val requesterPaysRole = gcsConfig.getString("requesterPaysRole")
       val projectTemplate = ProjectTemplate(projectOwners, projectEditors)
 
+      val notificationPubSubDAO = new org.broadinstitute.dsde.workbench.google.HttpGooglePubSubDAO(
```

Comment:
as a followup, I'd like to eliminate the rawls PubSubDAO implementation and just use the workbench-libs implementation everywhere. that'd make this a bit cleaner. but that's a separate issue